resources/js/Pages/Home.vue

```diff
@@ -45,183 +46,65 @@ const formattedDate = (
         .join(" ");
 
     return `${startDate} ${separator} ${endDate}`;
-};
+};*/
 </script>
 
 <template>
-    <AppLayout title="Home">
-        <InfoPopup v-if="$page.props.auth.user && canEnroll" />
-        <a
-            v-if="canEnroll"
-            href="#enroll-wrapper"
-            class="group fixed bottom-10 right-12 z-50 cursor-pointer border border-black bg-2023-red-dark p-2 px-3 font-bold text-white shadow shadow-2023-teal"
-        >
-            <OhVueIcon name="io-arrow-down" fill="white" />
-        </a>
-        <!-- LOGO & DATE -->
-        <section
-            class="relative flex min-h-screen flex-col content-center items-center justify-evenly gap-16 py-16"
-        >
-            <img
-                id="svg-1"
-                class="absolute left-[7%] w-36 animate-2023-maintenance-jump opacity-50 max-ml:hidden"
-                src="images/cy-sinf.svg"
-                alt="Stylized SINF logo"
-            />
-            <img
-                id="svg-2"
-                class="absolute right-[7%] top-16 w-32 animate-2023-maintenance-jump opacity-50 max-ml:hidden"
-                src="images/rc-sinf.svg"
-                alt="Stylized SINF logo"
-            />
-            <img
-                id="svg-3"
-                class="absolute bottom-24 right-[15%] w-24 animate-2023-maintenance-jump opacity-50 max-ml:hidden"
-                src="images/oc-sinf.svg"
-                alt="Stylized SINF logo"
-            />
-
-            <div class="relative">
-                <img
-                    class="w-96 max-ml:w-[300px]"
-                    src="images/sinf logo.png"
-                    alt="Stylized SINF logo"
-                />
-                <span
-                    class="margin-0 absolute -bottom-5 right-0 text-xl font-bold text-2023-teal"
-                    >{{ edition.year }}</span
-                >
+    <AppLayout title="Home" :show-navbar="false">
+        <div class="background">
```

Comment:
Currently, this div exceeds the total screen height which causes a scrollbar to appear, even when there is no need (we can fit the content on the normal browser window without scrolling). What we need to do is make this div `min-h-screen` as well as `flex` and `flex-col`. Then, below, use `grow` on the content `section` to make it fill the entirety of the flex parent.